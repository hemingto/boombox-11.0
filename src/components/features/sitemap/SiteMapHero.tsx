/**
 * @fileoverview Site map page hero section component
 * @source boombox-10.0/src/app/components/sitemap/sitemaphero.tsx
 * 
 * COMPONENT FUNCTIONALITY:
 * Displays the hero section for the sitemap page with a simple heading.
 * Uses semantic HTML and responsive padding following design system standards.
 * 
 * API ROUTES UPDATED:
 * - None (presentational component only)
 * 
 * DESIGN SYSTEM UPDATES:
 * - Uses existing responsive padding patterns (lg:px-16 px-6)
 * - Maintains semantic spacing with my-10 sm:my-20
 * - Semantic h1 element for proper heading hierarchy
 * 
 * @refactor Renamed file to PascalCase, added comprehensive documentation
 */

'use client';

import React from 'react';

/**
 * SiteMapHero component
 * 
 * Simple hero section for the sitemap page displaying the page title.
 * Uses responsive padding and spacing following the design system.
 * 
 * @example
 * ```tsx
 * <SiteMapHero />
 * ```
 */
export function SiteMapHero(): React.ReactElement {
  return (
    <div className="w-full my-10 sm:my-20 lg:px-16 px-6">
      <div>
        <h1 className="mb-4">Site map</h1>
      </div>
    </div>
  );
}

