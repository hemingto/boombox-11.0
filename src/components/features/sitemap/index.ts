/**
 * @fileoverview Sitemap feature components exports
 * @source boombox-10.0/src/app/components/sitemap/
 */

export { SiteMapHero } from './SiteMapHero';
export { SiteMapLinks } from './SiteMapLinks';

