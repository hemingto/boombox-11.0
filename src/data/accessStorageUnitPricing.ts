export const accessStorageUnitPricing = 45;

