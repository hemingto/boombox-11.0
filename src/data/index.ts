/**
 * @fileoverview Data exports for boombox-11.0
 * @refactor Centralized data exports for all static/configuration data
 */

// Customer Reviews
export * from './customerReviews';

// FAQ Data
export * from './faq';

// Insurance Options
export * from './insuranceOptions';

// Labor Options
export * from './laborOptions';

// Locations
export * from './locations';

// Bay Area Cities
export * from './bayareacities';

// Bay Area Service Area Coordinates
export * from './bayareaserviceareacoordinates';

// Popular Locations
export * from './popularlocations';

// Training Videos
export * from './trainingVideos';

// Sitemap Links
export * from './sitemapLinks';

// Storage Guidelines (contains JSX, uses .tsx extension)
export * from './storageGuidelines';

// Pricing Data
export * from './accessStorageUnitPricing';
export * from './cancelationpricing';
export * from './zipcodeprices';

