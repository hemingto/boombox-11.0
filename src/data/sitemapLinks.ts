/**
 * @fileoverview Sitemap navigation links data for sitemap page
 * @source boombox-10.0/src/app/components/sitemap/sitemaplinks.tsx (inline data)
 * @refactor Extracted sitemap data to centralized data folder for better maintainability
 */

/**
 * Individual sitemap link interface
 */
export interface SitemapLink {
  name: string;
  href: string;
}

/**
 * Sitemap section interface with category and links
 */
export interface SitemapSection {
  category: string;
  links: SitemapLink[];
}

/**
 * Sitemap navigation data organized by categories
 * 
 * Contains all public-facing pages organized into logical sections:
 * - General: Main navigation pages
 * - Blog Posts: Content articles
 * - Locations: Service area cities
 */
export const sitemapData: SitemapSection[] = [
  {
    category: 'General',
    links: [
      { name: 'Homepage', href: '/' },
      { name: 'Storage Unit Prices', href: '/storage-unit-prices' },
      { name: 'Locations', href: '/locations' },
      { name: 'Terms', href: '/terms' },
      { name: 'Careers', href: '/careers' },
      { name: 'Storage Guidelines', href: '/storage-guidelines' },
      { name: 'Holiday Storage', href: '/holiday-storage' },
      { name: 'Blog', href: '/blog' },
      { name: 'Storage Calculator', href: '/storage-calculator' },
      { name: 'Packing Supplies', href: '/packing-supplies' },
      { name: 'Insurance', href: '/insurance' },
      { name: 'How it works', href: '/how-it-works' },
      { name: 'Help Center', href: '/help-center' },
      { name: 'Driver Sign Up', href: '/driver-sign-up' },
      { name: 'Checklist', href: '/checklist' },
      { name: 'Get Quote', href: '/get-quote' },
    ],
  },
  {
    category: 'Blog Posts',
    links: [
      { name: 'Blog Post', href: '/blog-post-1' },
      { name: 'Blog Post', href: '/blog-post-2' },
      { name: 'Blog Post', href: '/blog-post-3' },
      { name: 'Blog Post', href: '/blog-post-4' },
      { name: 'Blog Post', href: '/blog-post-5' },
      // add more blog posts as needed...
    ],
  },
  {
    category: 'Locations',
    links: [
      { name: 'Burlingame', href: '/locations/burlingame' },
      { name: 'Cupertino', href: '/locations/cupertino' },
      { name: 'San Jose', href: '/locations/san-jose' },
      { name: 'Santa Clara', href: '/locations/santa-clara' },
      { name: 'Mountain View', href: '/locations/mountain-view' },
      { name: 'Palo Alto', href: '/locations/palo-alto' },
      { name: 'Berkeley', href: '/locations/berkeley' },
      // add more locations as needed...
    ],
  },
];

