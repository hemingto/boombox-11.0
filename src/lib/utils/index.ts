/**
 * @fileoverview Central utility exports for boombox-11.0
 * Organized by domain and functionality for clean imports
 */

// Core utility functions
export * from './cn';
export * from './phoneUtils';
export * from './currencyUtils';
export * from './navigationUtils';
export * from './emailUtils';

// Notification utilities
export * from './notificationUtils';
export * from './validationUtils';
export * from './formatUtils';
export * from './statusUtils';

// Blog utilities
export * from './blogUtils';

// Explicit exports to resolve conflicts
export {
  calculateMonthlyStorageCost,
  isInServiceArea,
  getServiceAreaForZip,
} from './businessUtils';

export {
  formatDateForInput,
  formatDateForDisplay,
  formatDateCompact,
  formatDateTime,
  formatTime,
  formatTime24Hour,
  formatTimestamp,
  parseAppointmentTime,
  calculateDeliveryWindow,
  getUnitSpecificStartTime,
  formatVerboseDate,
} from './dateUtils';

export {
  generateDriverToken,
  generateCustomerMoverChangeToken,
  notifyDriverReassignment,
  notifyCustomerMoverChange,
  recordDriverCancellation,
  recordMoverCancellation,
  type DriverCancellationData,
  type MoverCancellationData,
  type AvailableDriver,
} from './cancellationUtils';

// Domain-specific utilities (avoiding conflicts)
// Note: appointmentUtils contains parseLoadingHelpPrice which conflicts with pricingUtils
// Using pricingUtils version for parseLoadingHelpPrice
export {
  formatAppointmentTime,
  formatAppointmentDateForSms,
  formatTimeMinusOneHour,
  validateAppointmentDateTime,
  calculateAppointmentChanges,
  calculateFinalUnitCount,
  getUnitNumbersToRemove,
  getDayOfWeekForAvailability,
  validateMovingPartnerAvailability,
  generateDriverReconfirmToken,
  generateDriverWebViewUrl,
  formatDriverInfo,
  verifyTrackingToken,
  // parseLoadingHelpPrice - using pricingUtils version
} from './appointmentUtils';

// Explicit exports from availabilityUtils to avoid conflicts
export {
  DEFAULT_BUSINESS_HOURS,
  calculateDriverRequirements,
  formatTimeLocal,
  generateBusinessHourSlots,
  hasTimeConflict,
  isResourceAvailableInSlot,
  createTimeConflict,
  checkOnfleetTaskConflicts,
  determineAvailabilityLevel,
  getDistinctDaysOfWeekInMonth,
  groupBy,
  isDateInPast,
  generateCacheKey,
  getMinutesDifference,
  isValidTimeSlot,
  timeStringToMinutes,
  // getDayOfWeekString - conflicts with dateFormattingUtils, using that version
} from './availabilityUtils';

export * from './moverChangeUtils';

// Explicit exports from movingPartnerUtils to avoid conflicts
export {
  createDefaultMoverAvailability,
  parseTimeToMinutes,
  checkMoverExists,
  createMover,
  findAvailableMovingPartners,
  uploadFileToCloudinary,
  deleteOldCloudinaryFile,
  getMovingPartnerJobs,
  assignMovingPartnerDriver,
  // getDayOfWeekString - conflicts with other modules, already exported from dateFormattingUtils
} from './movingPartnerUtils';

// Messaging and communication utilities
// Note: twilioUtils contains server-side code (headers) - import directly in API routes only
export * from './messageClassificationUtils';
export * from './inboundMessageUtils';
// Note: dateFormattingUtils contains formatAppointmentTime which conflicts with appointmentUtils
// Using appointmentUtils version for appointment-specific formatting
// export * from './dateFormattingUtils';
// Explicit export of non-conflicting functions from dateFormattingUtils
export { formatAppointmentDate } from './dateFormattingUtils';

// Driver utilities (explicit exports to avoid conflicts)
export {
  getDriverTeamIds,
  createDefaultDriverAvailability,
  formatPhoneForOnfleet,
  mapVehicleTypeToOnfleet,
  buildOnfleetWorkerPayload,
  validateDriverUniqueness,
  findDriverInvitation,
  validateInvitationStatus,
  DEFAULT_DRIVER_SERVICES,
  DEFAULT_MOVING_PARTNER_VEHICLE_TYPE,
  DEFAULT_DRIVER_CAPACITY,
  DRIVER_STATUSES,
  type DriverRegistrationData,
  type OnfleetWorkerPayload,
  type DriverApprovalResult,
} from './driverUtils';

// Pricing utilities (explicit exports to avoid conflicts)
export {
  getBoomboxPriceByZipCode,
  formatStorageUnitPrice,
  formatInsurancePrice,
  parseLoadingHelpPrice, // Using pricingUtils version as canonical
} from './pricingUtils';

// Storage utilities (explicit exports)
export {
  getStorageUnitText,
} from './storageUtils';

// Sorting and pagination utilities
export * from './sortingUtils';

// Tracking utilities
export * from './trackingStatusUtils';
