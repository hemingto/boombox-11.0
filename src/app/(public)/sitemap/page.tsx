/**
 * @fileoverview Sitemap page - site navigation map
 * @source boombox-10.0/src/app/sitemap/page.tsx
 * @refactor Migrated to (public) route group with SEO metadata
 */

'use client';

import type { Metadata } from 'next';
import { SiteMapHero, SiteMapLinks } from '@/components/features/sitemap';
import { HelpCenterSection } from '@/components/features/landing';

// Note: Page is client component
// SEO metadata should be added via generateMetadata when converting to server component

export default function SiteMap() {
  return (
    <>
      <SiteMapHero />
      <SiteMapLinks />
      <HelpCenterSection />
    </>
  );
}

