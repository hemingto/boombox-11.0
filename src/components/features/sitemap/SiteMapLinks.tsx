/**
 * @fileoverview Site map navigation links component displaying categorized page links
 * @source boombox-10.0/src/app/components/sitemap/sitemaplinks.tsx
 * 
 * COMPONENT FUNCTIONALITY:
 * Displays all public-facing site pages organized by categories (General, Blog Posts, Locations).
 * Uses clickable link badges with hover states for easy navigation to any page on the site.
 * Data extracted to @/data/sitemapLinks for better maintainability.
 * 
 * API ROUTES UPDATED:
 * - None (presentational component only)
 * 
 * DESIGN SYSTEM UPDATES:
 * - bg-slate-100 → bg-surface-tertiary (semantic surface color)
 * - hover:bg-slate-200 → hover:bg-surface-secondary (semantic hover state)
 * - sm:active:bg-slate-300 → active:bg-surface-primary (semantic active state)
 * - Uses consistent responsive padding (lg:px-16 px-6)
 * - Maintains semantic spacing (mb-24 sm:mb-48)
 * 
 * @refactor Extracted data to centralized data folder, renamed to PascalCase, applied design system colors
 */

import React from 'react';
import Link from 'next/link';
import { sitemapData } from '@/data/sitemapLinks';
import type { SitemapSection } from '@/data/sitemapLinks';

/**
 * SiteMapLinks component
 * 
 * Displays a categorized list of all site pages as clickable link badges.
 * Each section contains a category heading and a grid of link badges.
 * 
 * @example
 * ```tsx
 * <SiteMapLinks />
 * ```
 */
export function SiteMapLinks(): React.ReactElement {
  return (
    <div className="lg:px-16 px-6 py-8 space-y-10 sm:mb-48 mb-24 max-w-7xl">
      {sitemapData.map((section: SitemapSection, index: number) => (
        <section key={index} aria-labelledby={`sitemap-category-${index}`}>
          <h2 id={`sitemap-category-${index}`} className="mb-4">
            {section.category}
          </h2>
          <nav aria-label={`${section.category} navigation`}>
            <div className="flex flex-wrap gap-4">
              {section.links.map((link, i) => (
                <Link
                  key={i}
                  href={link.href}
                  className="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
                  aria-label={`Navigate to ${link.name}`}
                >
                  {link.name}
                </Link>
              ))}
            </div>
          </nav>
        </section>
      ))}
    </div>
  );
}

