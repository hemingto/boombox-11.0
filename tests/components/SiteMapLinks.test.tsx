/**
 * @fileoverview Tests for SiteMapLinks component
 * Following boombox-11.0 testing standards (97â†’0 failure learnings)
 */
import { render, screen } from '@testing-library/react';
import { axe, toHaveNoViolations } from 'jest-axe';
import { testAccessibility } from '../utils/accessibility';
import { SiteMapLinks } from '@/components/features/sitemap/SiteMapLinks';
import { sitemapData } from '@/data/sitemapLinks';

expect.extend(toHaveNoViolations);

describe('SiteMapLinks', () => {
  // REQUIRED: Test cleanup
  beforeEach(() => {
    jest.clearAllMocks();
  });

  // REQUIRED: Basic rendering
  describe('Rendering', () => {
    it('renders without crashing', () => {
      render(<SiteMapLinks />);
      expect(screen.getByRole('heading', { level: 2, name: /general/i })).toBeInTheDocument();
    });

    it('renders all sitemap sections', () => {
      render(<SiteMapLinks />);
      
      // Should have 3 sections: General, Blog Posts, Locations
      const sections = screen.getAllByRole('region');
      expect(sections).toHaveLength(sitemapData.length);
    });

    it('renders section headings correctly', () => {
      render(<SiteMapLinks />);
      
      sitemapData.forEach((section) => {
        const heading = screen.getByRole('heading', { level: 2, name: section.category });
        expect(heading).toBeInTheDocument();
      });
    });

    it('renders all navigation links', () => {
      render(<SiteMapLinks />);
      
      // Count total links across all sections
      const totalLinks = sitemapData.reduce((sum, section) => sum + section.links.length, 0);
      const links = screen.getAllByRole('link');
      expect(links).toHaveLength(totalLinks);
    });

    it('renders links with correct hrefs', () => {
      render(<SiteMapLinks />);
      
      // Check a few specific links
      expect(screen.getByRole('link', { name: /Navigate to Homepage/i })).toHaveAttribute('href', '/');
      expect(screen.getByRole('link', { name: /Navigate to Get Quote/i })).toHaveAttribute('href', '/get-quote');
      expect(screen.getByRole('link', { name: /Navigate to Locations$/i })).toHaveAttribute('href', '/locations');
    });
  });

  // MANDATORY: Accessibility testing
  describe('Accessibility', () => {
    it('has no accessibility violations', async () => {
      const renderResult = render(<SiteMapLinks />);
      await testAccessibility(renderResult);
    });

    it('uses semantic HTML with proper section structure', () => {
      render(<SiteMapLinks />);
      
      // Each category should be a section
      const sections = screen.getAllByRole('region');
      expect(sections.length).toBeGreaterThan(0);
    });

    it('has proper heading hierarchy', () => {
      const { container } = render(<SiteMapLinks />);
      
      // All category headings should be h2
      const h2Headings = container.querySelectorAll('h2');
      expect(h2Headings.length).toBe(sitemapData.length);
    });

    it('links have descriptive aria-labels', () => {
      render(<SiteMapLinks />);
      
      const firstLink = screen.getByRole('link', { name: /Navigate to Homepage/i });
      expect(firstLink).toHaveAttribute('aria-label', 'Navigate to Homepage');
    });

    it('each section has proper aria-labelledby', () => {
      const { container } = render(<SiteMapLinks />);
      
      const sections = container.querySelectorAll('section');
      sections.forEach((section, index) => {
        expect(section).toHaveAttribute('aria-labelledby', `sitemap-category-${index}`);
      });
    });

    it('navigation elements have proper aria-labels', () => {
      render(<SiteMapLinks />);
      
      // Check for navigation landmarks
      const navElements = screen.getAllByRole('navigation');
      expect(navElements.length).toBeGreaterThan(0);
      
      // First nav should be for General
      expect(navElements[0]).toHaveAttribute('aria-label', 'General navigation');
    });
  });

  // Design System Integration
  describe('Design System Compliance', () => {
    it('applies responsive padding classes to container', () => {
      const { container } = render(<SiteMapLinks />);
      const outerDiv = container.firstChild as HTMLElement;
      
      expect(outerDiv).toHaveClass('lg:px-16');
      expect(outerDiv).toHaveClass('px-6');
    });

    it('applies responsive vertical spacing', () => {
      const { container } = render(<SiteMapLinks />);
      const outerDiv = container.firstChild as HTMLElement;
      
      expect(outerDiv).toHaveClass('sm:mb-48');
      expect(outerDiv).toHaveClass('mb-24');
    });

    it('uses semantic surface colors for link badges', () => {
      render(<SiteMapLinks />);
      
      const firstLink = screen.getByRole('link', { name: /Navigate to Homepage/i });
      expect(firstLink).toHaveClass('bg-surface-tertiary');
      expect(firstLink).toHaveClass('hover:bg-surface-secondary');
    });

    it('applies consistent spacing between sections', () => {
      const { container } = render(<SiteMapLinks />);
      const outerDiv = container.firstChild as HTMLElement;
      
      expect(outerDiv).toHaveClass('space-y-10');
    });

    it('maintains max-width constraint', () => {
      const { container } = render(<SiteMapLinks />);
      const outerDiv = container.firstChild as HTMLElement;
      
      expect(outerDiv).toHaveClass('max-w-7xl');
    });
  });

  // Link Styling & Behavior
  describe('Link Styling', () => {
    it('applies correct styling to link badges', () => {
      render(<SiteMapLinks />);
      
      const links = screen.getAllByRole('link');
      links.forEach((link) => {
        expect(link).toHaveClass('text-sm');
        expect(link).toHaveClass('py-2');
        expect(link).toHaveClass('px-4');
        expect(link).toHaveClass('rounded-full');
        expect(link).toHaveClass('cursor-pointer');
      });
    });

    it('includes transition for smooth hover effects', () => {
      render(<SiteMapLinks />);
      
      const firstLink = screen.getByRole('link', { name: /Navigate to Homepage/i });
      expect(firstLink).toHaveClass('transition-colors');
      expect(firstLink).toHaveClass('duration-150');
    });

    it('has active state styling', () => {
      render(<SiteMapLinks />);
      
      const firstLink = screen.getByRole('link', { name: /Navigate to Homepage/i });
      expect(firstLink).toHaveClass('active:bg-surface-primary');
    });
  });

  // Data Integration
  describe('Data Integration', () => {
    it('renders data from imported sitemapData', () => {
      render(<SiteMapLinks />);
      
      // Verify first category from data
      expect(screen.getByRole('heading', { level: 2, name: sitemapData[0].category })).toBeInTheDocument();
    });

    it('renders all links from each category', () => {
      render(<SiteMapLinks />);
      
      // Check that all links from first category are present
      const firstCategory = sitemapData[0];
      const allLinks = screen.getAllByRole('link');
      
      // Verify each link from first category exists
      firstCategory.links.forEach((link) => {
        const matchingLink = allLinks.find((el) => 
          el.getAttribute('href') === link.href
        );
        expect(matchingLink).toBeDefined();
        expect(matchingLink).toHaveTextContent(link.name);
      });
    });

    it('uses flexbox layout for link badges', () => {
      const { container } = render(<SiteMapLinks />);
      
      const linkContainers = container.querySelectorAll('.flex.flex-wrap');
      expect(linkContainers.length).toBe(sitemapData.length);
      
      linkContainers.forEach((container) => {
        expect(container).toHaveClass('gap-4');
      });
    });
  });

  // Component Structure
  describe('Component Structure', () => {
    it('wraps each category in a section element', () => {
      const { container } = render(<SiteMapLinks />);
      
      const sections = container.querySelectorAll('section');
      expect(sections).toHaveLength(sitemapData.length);
    });

    it('includes nav elements for each category', () => {
      render(<SiteMapLinks />);
      
      const navElements = screen.getAllByRole('navigation');
      expect(navElements).toHaveLength(sitemapData.length);
    });

    it('applies margin bottom to category headings', () => {
      render(<SiteMapLinks />);
      
      const headings = screen.getAllByRole('heading', { level: 2 });
      headings.forEach((heading) => {
        expect(heading).toHaveClass('mb-4');
      });
    });
  });

  // Snapshot Testing
  describe('Snapshot', () => {
    it('matches snapshot', () => {
      const { container } = render(<SiteMapLinks />);
      expect(container).toMatchSnapshot();
    });
  });
});

