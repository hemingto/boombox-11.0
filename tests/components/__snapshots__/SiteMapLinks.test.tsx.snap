// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`SiteMapLinks Snapshot matches snapshot 1`] = `
<div>
  <div
    class="lg:px-16 px-6 py-8 space-y-10 sm:mb-48 mb-24 max-w-7xl"
  >
    <section
      aria-labelledby="sitemap-category-0"
    >
      <h2
        class="mb-4"
        id="sitemap-category-0"
      >
        General
      </h2>
      <nav
        aria-label="General navigation"
      >
        <div
          class="flex flex-wrap gap-4"
        >
          <a
            aria-label="Navigate to Homepage"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/"
          >
            Homepage
          </a>
          <a
            aria-label="Navigate to Storage Unit Prices"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/storage-unit-prices"
          >
            Storage Unit Prices
          </a>
          <a
            aria-label="Navigate to Locations"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/locations"
          >
            Locations
          </a>
          <a
            aria-label="Navigate to Terms"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/terms"
          >
            Terms
          </a>
          <a
            aria-label="Navigate to Careers"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/careers"
          >
            Careers
          </a>
          <a
            aria-label="Navigate to Storage Guidelines"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/storage-guidelines"
          >
            Storage Guidelines
          </a>
          <a
            aria-label="Navigate to Holiday Storage"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/holiday-storage"
          >
            Holiday Storage
          </a>
          <a
            aria-label="Navigate to Blog"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/blog"
          >
            Blog
          </a>
          <a
            aria-label="Navigate to Storage Calculator"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/storage-calculator"
          >
            Storage Calculator
          </a>
          <a
            aria-label="Navigate to Packing Supplies"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/packing-supplies"
          >
            Packing Supplies
          </a>
          <a
            aria-label="Navigate to Insurance"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/insurance"
          >
            Insurance
          </a>
          <a
            aria-label="Navigate to How it works"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/how-it-works"
          >
            How it works
          </a>
          <a
            aria-label="Navigate to Help Center"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/help-center"
          >
            Help Center
          </a>
          <a
            aria-label="Navigate to Driver Sign Up"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/driver-sign-up"
          >
            Driver Sign Up
          </a>
          <a
            aria-label="Navigate to Checklist"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/checklist"
          >
            Checklist
          </a>
          <a
            aria-label="Navigate to Get Quote"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/get-quote"
          >
            Get Quote
          </a>
        </div>
      </nav>
    </section>
    <section
      aria-labelledby="sitemap-category-1"
    >
      <h2
        class="mb-4"
        id="sitemap-category-1"
      >
        Blog Posts
      </h2>
      <nav
        aria-label="Blog Posts navigation"
      >
        <div
          class="flex flex-wrap gap-4"
        >
          <a
            aria-label="Navigate to Blog Post"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/blog-post-1"
          >
            Blog Post
          </a>
          <a
            aria-label="Navigate to Blog Post"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/blog-post-2"
          >
            Blog Post
          </a>
          <a
            aria-label="Navigate to Blog Post"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/blog-post-3"
          >
            Blog Post
          </a>
          <a
            aria-label="Navigate to Blog Post"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/blog-post-4"
          >
            Blog Post
          </a>
          <a
            aria-label="Navigate to Blog Post"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/blog-post-5"
          >
            Blog Post
          </a>
        </div>
      </nav>
    </section>
    <section
      aria-labelledby="sitemap-category-2"
    >
      <h2
        class="mb-4"
        id="sitemap-category-2"
      >
        Locations
      </h2>
      <nav
        aria-label="Locations navigation"
      >
        <div
          class="flex flex-wrap gap-4"
        >
          <a
            aria-label="Navigate to Burlingame"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/locations/burlingame"
          >
            Burlingame
          </a>
          <a
            aria-label="Navigate to Cupertino"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/locations/cupertino"
          >
            Cupertino
          </a>
          <a
            aria-label="Navigate to San Jose"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/locations/san-jose"
          >
            San Jose
          </a>
          <a
            aria-label="Navigate to Santa Clara"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/locations/santa-clara"
          >
            Santa Clara
          </a>
          <a
            aria-label="Navigate to Mountain View"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/locations/mountain-view"
          >
            Mountain View
          </a>
          <a
            aria-label="Navigate to Palo Alto"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/locations/palo-alto"
          >
            Palo Alto
          </a>
          <a
            aria-label="Navigate to Berkeley"
            class="bg-surface-tertiary text-sm py-2 px-4 rounded-full cursor-pointer hover:bg-surface-secondary active:bg-surface-primary transition-colors duration-150"
            href="/locations/berkeley"
          >
            Berkeley
          </a>
        </div>
      </nav>
    </section>
  </div>
</div>
`;
